api-tester/src/main/java/com/revenuecat/apitester/kotlin/StoreProductAPI.kt

```diff
@@ -1,30 +1,20 @@
 package com.revenuecat.apitester.kotlin
 
 import com.revenuecat.purchases.ProductType
+import com.revenuecat.purchases.models.Price
 import com.revenuecat.purchases.models.StoreProduct
 import org.json.JSONObject
 
 @Suppress("unused", "UNUSED_VARIABLE")
 private class StoreProductAPI {
     fun check(product: StoreProduct) {
         with(product) {
-            val sku: String = sku
+            val storeProductId: String = storeProductId
             val type: ProductType = type
-            val price: String = price
-            val priceAmountMicros: Long = priceAmountMicros
-            val priceCurrencyCode: String = priceCurrencyCode
-            val originalPrice: String? = originalPrice
-            val originalPriceAmountMicros: Long = originalPriceAmountMicros
+            val oneTimeProductPrice: Price? = oneTimeProductPrice
```

Comment:
so this'll be null for subs?
I think the iOS equivalent is showing the price of the base plan, would it be achievable here?

Reply:
definitely achievable.. we went back and forth on that a lot. we thought pulling the base plan price might be confusing to customers.

IMO i think it depends on the default purchase behavior, which we are still determining. perhaps we pull the price from whatever we deem "best offer", or base plan if we purchase that by default. that way, whatever price shows up in the storeproduct is the default purchase price

i'm starting a list of open design questions, i'll be sure to add this.

Reply:
> we thought pulling the base plan price might be confusing to customers.

I think the main issue is devs that upgrade SDK version and don't change their price logic. They will be showing `.price` thinking the free trial will be automatically applied without realizing that they will always be showing base plan price.

I think we can iterate over this though and we can have more discussions and change it later, but I wouldn't block this PR deciding what we do here. I like the `oneTimeProductPrice` for now since that's what Google is doing